test/irmin-pack/test_lower.ml

```diff
@@ -162,10 +159,9 @@ module Store_tc = struct
 
   let test_dir = "_build"
 
-  let mkdir_if_needed path =
-    match Io.mkdir path with
-    | Error (`File_exists _) | Ok () -> Ok ()
-    | _ as r -> r
+  let mkdir path =
+    Common.rm_dir path;
+    Io.mkdir path
```

Comment:
It was originally coded this way to give control of unlinking paths, but this pattern works since I think `unlink_lower` in `init` isn't used in the tests. Could you also remove `unlink_lower` if that is the case?

Reply:
Ha I missed `?unlink_lower` at the time, and have removed it now :) For context, rerunning the tests led to different results and so I thought it would be more convenient to always start from a clean slate.

Reply:
Yeah, I definitely think it is better to start clean. Not sure what was going through my mind originally. :sweat_smile: